feat(tools): add team reviewers

naomi-lgbt opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #XXXXX

<!-- Feel free to add any additional description of changes below this line -->

This ties in to #41591, adding another optional parameter to the workflow to request team reviews (since this is a different value than reviewers in the API).

@raisedadead In order to leave this tool as universal as possible, I thought it would be better to add another parameter instead of renaming the existing one - if you agree, then we'd need to update #41591 to pass `team_reviewers` instead of `reviewers`.

@RandellDawson would you mind giving this a sanity check as well?

Commit:
feat(tools): add team reviewers

Adds the team_reviewers option to the action, to allow for requesting
reviews from an org team (or from specific members, still).

raisedadead reviewed (commented):
Thanks for making this quick change, I think this is much better. BTW if I understand the spec correctly the format for the team_reviewers is a slug:

Team: https://github.com/orgs/freeCodeCamp/teams/dev-team
Slug: `dev-team`

So we would need the config definition to mention a quick example in comments for future reference.

Commit:
fix(tools): add comment

Add a comment to the action.yml documenting that the team_reviewers
takes a slug, not a full @-mention.

raisedadead reviewed (approved):
Let's give this a go

ojeytonwilliams reviewed (approved):
LGTM :+1:
